src/applications/representative-appoint/prefill-transformer.js

```diff
@@ -0,0 +1,68 @@
+import { prefillBankInformation } from 'platform/forms-system/src/js/definitions/directDeposit';
```

Comment:
Are you going to clean this file up later with the stuff we don't need? I'm noticing bank information which is not part of the form.

Reply:
Same question applies to the schema and fixtures files.